widget/embedded/src/store/notification.ts

```diff
@@ -67,7 +68,13 @@ export const useNotificationStore = createSelectors(
         getNotifications: () => {
           const { isSynced, notifications } = get();
           return isSynced
-            ? notifications.sort((a, b) => b.creationTime - a.creationTime)
+            ? notifications.sort((a, b) =>
+                !a.transactionCreationTime ||
+                !b.transactionCreationTime ||
+                a.transactionCreationTime === b.transactionCreationTime
+                  ? b.creationTime - a.creationTime
+                  : b.transactionCreationTime - a.transactionCreationTime
+              )
```

Comment:
We know that if the `transactionCreationTime` property exists, the transaction is newer than an item that does not have this property. Therefore, we can change this to:

```ts
notifications.sort((a, b) => {
      if (a.transactionCreationTime && !b.transactionCreationTime) {
        return -1;
      }
      if (!a.transactionCreationTime && b.transactionCreationTime) {
        return 1;
      }
      if (!a.transactionCreationTime && !b.transactionCreationTime) {
        return b.creationTime - a.creationTime;
      }
      return b.transactionCreationTime - a.transactionCreationTime;
    })
```

Reply:
Tnx Fixed